### PR TITLE
B4 Timeline

### PR DESCRIPTION
**Pull request by Gravill0n**
Showing B4 Project (data from Nantes' intranet) with the close end of the C++ pool

### CODE REVIEW

File: epitech2021-timeline.js [hunk 1]
```diff
@@ -29,7 +29,35 @@ function drawChart() {
...
+    [ 'B4 - Mathematics', '201yams', sdate(12, 02, 2018), edate(25, 02, 2018)],
+    [ 'B4 - Mathematics', '202invendus', sdate(26, 02, 2018), edate(11, 03, 2018)],
+    [ 'B4 - Mathematics', '203hotline', sdate(12, 03, 2018), edate(25, 03, 2018)],
+    [ 'B4 - Mathematics', '204canards', sdate(26, 03, 2018), edate(08, 04, 2018)],
+    [ 'B4 - Mathematics', '205QI', sdate(09, 04, 2018), edate(22, 04, 2018)],
+    [ 'B4 - Mathematics', '206neutrinos', sdate(23, 04, 2018), edate(06, 05, 2018)],
+    [ 'B4 - Mathematics', '207demographie', sdate(07, 05, 2018), edate(20, 05, 2018)],
+    [ 'B4 - Mathematics', '208chevillettes', sdate(14, 05, 2018), edate(27, 05, 2018)],
+    [ 'B4 - Mathematics', '209sondage', sdate(21, 05, 2018), edate(03, 06, 2018)],
+    [ 'B4 - Shell Scripting ', 'ShellScrip Project', sdate(22, 01, 2018), edate(28, 01, 2018)],
```
Comment (ghost) [inline epitech2021-timeline.js]: Fix Typo :
ShellScrip Project -> ShellScrip**t** Project 😃

**Comment by demaisj** [comment]
First of all, thank you for your contribution.
Where did you get the data for semester 4 on the intranet ? I couldn't find any information on my side (no activities, no modules, no planning, event in other cities).

**Review by demaisj — CHANGES_REQUESTED** [review]
To make the timeline easier to read, each modules should be ordered by the starting date of their first project in ascending order. (C++ part1 & c++ part2 are considered as two different modules for example)

**Comment by Gravill0n** [comment]
Hello,


Yes, I got it on the Nantes's Intranet, we have been switched to the Semester 4 Monday, so we have been able to register for the modules.


Pierre.

________________________________
De : Jordan Demaison <notifications@github.com>
Envoyé : jeudi 18 janvier 2018 19:18:04
À : demaisj/Epitech-2021-Timeline
Cc : pierre pelloquin; Author
Objet : Re: [demaisj/Epitech-2021-Timeline] B4 Timeline (#4)


First of all, thank you for your contribution.
Where did you get the data for semester 4 on the intranet ? I couldn't find any information on my side (no activities, no modules, no planning, event in other cities).

**Review by demaisj — APPROVED** [review]
This is OK, I'm merging this pull request. Once again thank you for your contribution.

**Pull request merged**